__tests__/app.test.js

```diff
@@ -0,0 +1,34 @@
+const app = require("../app")
+const db = require('../db/connection')
+const seed = require('../db/seeds/seed')
+const testData = require('../db/data/test-data/index')
+const request = require('supertest')
+
+beforeEach(() => {
+    return seed(testData)
+})
+
+afterAll(() => {
+    db.end()
+})
+
+describe('/api/categories', () => {
+    describe('GET requests', () => {
+        describe('GET /api/categories', () => {
+            test('happy path', () => {
+                return request(app).get('/api/categories').expect(200).then(({body}) => {
+                    expect(Array.isArray(body)).toBe(true)
+                    body.forEach(object => {
```

Comment:
Make sure any time we do a test on an array, we also make sure we are checking the correct length, a forEach test on an empty array will always pass so its important we check that we are at least not testing an empty array

Reply:
Thanks for your help! Shall I just merge it with those changes or submit another request?

Reply:
This is good to go!